plumbing/format/index/index.go

```diff
@@ -93,8 +93,8 @@ type TreeEntry struct {
 	Hash plumbing.Hash
 }
 
-// ResolveUndo when a conflict is resolved (e.g. with "git add path"), these
-// higher stage entries will be removed and a stage-0 entry with proper
+// ResolveUndo is used when a conflict is resolved (e.g. with "git add path"),
+// these higher stage entries will be removed and a stage-0 entry with proper
```

Comment:
The phrases in this paragraph have mixed tenses (present and future) and does not make sense, rewrite the paragraph.